mlir/lib/Gradient/IR/GradientDialect.cpp

```diff
@@ -20,6 +22,23 @@ using namespace catalyst::gradient;
 
 #include "Gradient/IR/GradientOpsDialect.cpp.inc"
 
+//===----------------------------------------------------------------------===//
+// Complex Dialect Interfaces
```

Comment:
"Complex" :) What about,
```suggestion
// Interfaces.
```

Reply:
Ah thanks, copy paste mistake 😅